src/odemis/gui/cont/tabs/tab_bar_controller.py

```diff
@@ -0,0 +1,188 @@
+# -*- coding: utf-8 -*-
+
+"""
+@author: Rinze de Laat, Éric Piel, Philip Winkler, Victoria Mavrikopoulou,
+         Anders Muskens, Bassim Lazem
+
+Copyright © 2012-2022 Rinze de Laat, Éric Piel, Delmic
+
+Handles the switch of the content of the main GUI tabs.
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License version 2 as published by the Free
+Software Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+
+"""
+
+import logging
+import wx
+# IMPORTANT: wx.html needs to be imported for the HTMLWindow defined in the XRC
+# file to be correctly identified. See: http://trac.wxwidgets.org/ticket/3626
+# This is not related to any particular wxPython version and is most likely permanent.
+import wx.html
```

Comment:
This only needs to be done *once* somewhere in Odemis before loading a XRC file. 
I'd suggest to move it somewhere else. Maybe `gui.main.py` is the best place. Alternatively, we could "hide" it a little bit more by placing it in `gui.xmlh.__init__.py`. I don't think it helps to hide things though...

Reply:
Might be good to add to clarify in the comment that this only needs to be done once somewhere in Odemis before loading a XRC file.